hmt_escrow/storage.py

```diff
@@ -165,4 +238,4 @@ def _encrypt(public_key: bytes, msg: str) -> bytes:
     import doctest
     from test_manifest import manifest
     from job import Job
-    doctest.testmod(raise_on_error=True)
```

Comment:
dont forget about this. also there must be another way to do this on the cli so we dont have to change code every time